backend/src/bundles/users/user.repository.ts

```diff
@@ -9,8 +9,13 @@ class UserRepository implements Repository {
         this.userModel = userModel;
     }
 
-    public find(): ReturnType<Repository['find']> {
-        return Promise.resolve(null);
+    public async find(payload: string | number): Promise<UserEntity | null> {
```

Comment:
find method should only find by id.

Please create a specific method if you want to use different fields

for example findByEmail()